package.json

```diff
@@ -7,10 +7,10 @@
     "node": "10.11.0"
   },
   "scripts": {
-    "test": "npm run db-init && nyc mocha test/**/**/**.test.js --require @babel/register --require @babel/polyfill --exit",
-    "start": "npm run clean && npm run build && node dist/app.js",
+    "test": "npm run db-init && nyc mocha test/**/**/**.test.js --require babel-register --require babel-polyfill --exit",
+    "start": "npm run build && node --require babel-polyfill --require babel-register dist/app.js",
```

Comment:
If our babel configurations are transpiling the files correctly then I do not think we still have to start our app with babel. This does not feel like a long term solution so I dont think we should merge this particular changes in. It's like transpiling and still starting the app with babel-node.

Reply:
You are right. Let me fix that.